tendermint/src/rpc/error.rs

```diff
@@ -179,6 +195,7 @@ impl From<Code> for i32 {
             Code::InvalidParams => -32602,
             Code::InternalError => -32603,
             Code::ServerError => -32000,
+            Code::WebSocketError => 0,
```

Comment:
This is the same as HttpError 🤔 And in `impl From<i32> for Code` it will also be interpreted as such. Is that on purpose?

Reply:
I don't understand the Error code system. It is similar to an HTTP error?

Reply:
The error codes are transferred over the wire to signal a certain error type occured. Strings aren't really helpful here and integers work for every language and over JSON. As far as I can see, there is no special WebSocketError on the tendermint go side: https://github.com/tendermint/tendermint/blob/976eebe281685cd7982e03a24ce06ab3ff0561e8/rpc/jsonrpc/types/types.go#L223-L241